.github/workflows/main.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
       - name: Run linters
-        uses: golangci/golangci-lint-action@v6
+        uses: golangci/golangci-lint-action@v5
```

Comment:
this should be 8. It definitely shouldn't be a downgrade, the ci.yaml uses v6, I update these to v8. Maybe we should update baton-template